server/src/handler/mod.rs

```diff
@@ -16,6 +19,18 @@ impl<Deps: TpmContextDeps> CommandHandler<Deps> {
     pub fn new(crypto: Deps::Crypto) -> Self {
         Self { crypto }
     }
+    /// invariant: buffer will fit into TPM2B_DIGEST which is much less
+    /// than the maximum request of SP 800 90A
+    fn infallable_get_random(&mut self, buffer: &mut [u8]) {
+        if self.crypto.fill_bytes(&[], buffer).is_err() {
+            let mut entropy = <Deps::Crypto as Drbg>::Entropy::default();
```

Comment:
I don't love that part of re-setting up this DRBG is leaking into this call, via manually calling `default` and `fill_entropy` and `reseed`. This feels like it should be a function on the type to me.

I feel similarly about this function in question, why is this not implemented on `Drbg`?

Reply:
I am not following what you mean, Entropy is a slow hardware based source of randomness, it is very high quality source of entropy. DRBG is just rng algorithm that requires reseeding every once in a while. They are two separate things.